test/extended/image_ecosystem/plugin.go

```diff
@@ -770,6 +771,65 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 
 		})
 
+		g.It("jenkins-plugin test imagestream SCM", func() {
+			jobName := "test-imagestream-scm"
+			g.By("creating a jenkins job with an imagestream SCM")
+			data := j.readJenkinsJob("imagestream-scm-job.xml", oc.Namespace())
+			j.createItem(jobName, data)
+
+			// Because polling is enabled, a job should start automatically and fail
+			// Wait long enough for nothing to be running
+			time.Sleep(3 * time.Minute)
```

Comment:
Perhaps a wait.Poll{ getResource(...) } on the current job colors could eliminate the static wait?
http://stackoverflow.com/questions/14843874/from-jenkins-how-do-i-get-a-list-of-the-currently-running-jobs-in-json